python/ql/src/Security/CWE-116/BadTagFilter.qhelp

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE qhelp PUBLIC
+  "-//Semmle//qhelp//EN"
+  "qhelp.dtd">
+<qhelp>
+
+<overview>
+<p>
+Parsing general HTML using regular expressions is impossible, however it is possible to match
+single HTML tags. However, if the regexp is not written well it might be easy 
+to circumvent the regexp, which can lead to XSS or other security issues.
```

Comment:
Nit: I don't think it came across very clearly that the important bit of the first sentence is about the matching of a single HTML tag with a regexp. This was my best attempt for making it more clear, but feel free to do it any other way, or just ignoring it.

```suggestion
It is possible to match single HTML tags using regular expressions (parsing general HTML using 
regular expressions is impossible) . However, if the regexp is not written well it might be easy 
to circumvent the regexp, which can lead to XSS or other security issues.
```

Reply:
I update the qhelp inspired by your suggestion. 
(I also did two other small changes).